.github/workflows/deploy.yml

```diff
@@ -0,0 +1,45 @@
+name: "Test typescript-action"
```

Comment:
thanks for adding this example! for the time being, however, let's remove this workflow - I have #2 open to set up some proper continuous checks for the project, which I'm hoping will be a suite of more easily runnable checks that can be used to verify specific parts of the code.